OasisDEX/oasis-borrow
# 🧿 Ajna overview section modals and tooltips

# Ajna overview section modals and tooltips

Covers:
https://app.shortcut.com/oazo-apps/story/10311/update-ltv-modal
https://app.shortcut.com/oazo-apps/story/9254/earn-add-earn-tooltips
https://app.shortcut.com/oazo-apps/story/9288/update-borrow-tooltips-for-oasis-x-ajna-rewards
https://app.shortcut.com/oazo-apps/story/10300/earn-update-current-earnings-tooltip

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/dbb8cb96-d7b3-4ba2-864e-06a05436f64c)
  
## Changes 👷‍♀️

- Added ability to display modal in overview section footer items, the same as for overview content cards,
- added modals to Ajna Borrow and Earn in overview section footers,
- updated some of the Ajna Borrow and Earn content card modals,
- added ✨ tooltip to borrow rate in Ajna Borrow overview section footer.
  
## How to test 🧪

- Check all the modals in both content cards and footer section in Borrow and Earn UI,
- check borrow rate tooltip.
